CMake/SlicerExtensionCPack.cmake

```diff
@@ -59,6 +59,14 @@ endforeach()
 include(SlicerMacroExtractRepositoryInfo)
 SlicerMacroExtractRepositoryInfo(VAR_PREFIX ${EXTENSION_NAME})
 
+#-----------------------------------------------------------------------------
+# Set extension revision
+#-----------------------------------------------------------------------------
+if(NOT "${${EXTENSION_NAME}_FORCED_REVISION}" STREQUAL "")
```

Comment:
Originally the CMake configuration output only contained `Skipping repository info extraction: directory` message twice, which was not very helpful. After adding `${MY_VAR_PREFIX}` to the `AUTHOR_WARNING` message I got these two more informative messages:
- `Skipping Sandbox repository info extraction: directory`
- `Skipping EXTENSION repository info extraction: directory`

I think we should add `${MY_VAR_PREFIX}` to the warning message.